src/onset.jl

```diff
@@ -2,21 +2,41 @@
 # Types
 #---------------
 
+"""
+    struct UniformOnset <: AbstractOnset
+Provides a Uniform Distribution in the inter-event-distances.
+`width`  is the width of the uniform distribution (=> the jitter)
+`offset` is the minimal distance. The maximal distance is `offset + width`
```

Comment:
```suggestion
Provide a Uniform Distribution of the inter-event-distances.
`width`  is the width of the uniform distribution (=> the jitter). Since the lower bound is 0, `width` is also the upper bound.
`offset` is the minimal distance. The maximal distance is `offset + width`.
```